index.html

```diff
@@ -12894,9 +12894,8 @@ <h3>Serializing/Deserializing RDF</h3>
             <code>U+005C</code> (<code>\</code>) and <code>U+0022</code> (<code>"</code>)
             which SHOULD be serialized as <code>\\</code> and <code>\"</code> respectively.</li>
         </ul>
-        <div class="issue changed">The JSON Canonicalization Scheme [[?JCS]]
-          is an emerging standard for JSON canonicalization
-          not yet ready to be referenced.
+        <div class="issue changed">The JSON Canonicalization Scheme [[RFC8785]]
+          is a standard for JSON canonicalization.
```

Comment:
This rewording now doesn't flow well with the text below.  How much can we adjust it at this point due to this recent publication?  Seems like a minimal change would be something like:
```diff
-When a JSON canonicalization standard becomes available,
-this specification will likely be updated to require such a canonical representation.
+This specification will likely be updated to require such a canonical representation.
```

Reply:
I didn't want to change too much, but I did some minor changes. The group can re-address post publication, but @iherman can be the judge. Really, the whole informative section should be replaced with a link to the RFC.

Reply:
I did make the change you suggested, though.